api/v1alpha08/sonataflow_types.go

```diff
@@ -16,14 +16,28 @@ package v1alpha08
 
 import (
 	cncfmodel "github.com/serverlessworkflow/sdk-go/v2/model"
-	v1 "k8s.io/api/core/v1"
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"knative.dev/pkg/apis"
 	duckv1 "knative.dev/pkg/apis/duck/v1"
 
 	"github.com/kiegroup/kogito-serverless-operator/api"
 )
 
+// FlowPodSpecTemplate describes the desired custom Kubernetes PodSpecTemplate definition for the deployed flow.
+//
+// The FlowContainer describes the container where the actual flow is running. It will override any default definitions.
+// For example, to override the image one can use `.spec.podSpecTemplate.flowContainer.image = my/image:tag`.
+type FlowPodSpecTemplate struct {
+	// FlowContainer is the Kubernetes container where the workflow application should run.
+	// One can change this attribute in order to override the defaults provided by the operator.
+	// +optional
+	FlowContainer   corev1.Container `json:"flowContainer,omitempty"`
```

Comment:
When FlowContainer is specified, will the build state be skipped?

Reply:
Yes! See my last PR.